platform/ui/src/components/userPreferencesForm/UserPreferencesForm.js

```diff
@@ -63,30 +40,35 @@ function UserPreferencesForm({
   onSave,
   onResetToDefaults,
   windowLevelData,
-  hotkeyDefinitions,
   generalPreferences,
-  hotkeysManager,
   defaultLanguage,
   hotkeyDefaults,
+  hotkeyDefinitions,
+  hotkeyRecord,
 }) {
-  const [tabs, setTabs] = useState(createTabs());
-
-  const createTabsState = (
-    windowLevelData,
-    hotkeyDefinitions,
-    generalPreferences
-  ) => {
-    return {
-      Hotkeys: { hotkeyDefinitions },
-      'Window Level': { windowLevelData },
-      General: { generalPreferences },
-    };
-  };
-
-  const [tabsState, setTabsState] = useState(
-    createTabsState(windowLevelData, hotkeyDefinitions, generalPreferences)
-  );
-
+  const [tabs, setTabs] = useState([
```

Comment:
Looks like setTabs is not being used, this should be a component state?

Reply:
I've changed this and now its just a reference.